apps/explorer/src/app/components/markets/market-details.tsx

```diff
@@ -39,133 +39,79 @@ export const MarketDetails = ({ market }: { market: MarketInfoWithData }) => {
     return [];
   };
 
-  const oraclePanels = isEqual(
+  const showTwoOracles = isEqual(
     getSigners(settlementData),
     getSigners(terminationData)
-  )
-    ? [
-        {
-          title: t('Settlement Oracle'),
-          content: (
-            <OracleInfoPanel
-              noBorder={false}
-              market={market}
-              type="settlementData"
-            />
-          ),
-        },
-        {
-          title: t('Termination Oracle'),
-          content: (
-            <OracleInfoPanel
-              noBorder={false}
-              market={market}
-              type="termination"
-            />
-          ),
-        },
-      ]
-    : [
-        {
-          title: t('Oracle'),
-          content: (
-            <OracleInfoPanel
-              noBorder={false}
-              market={market}
-              type="settlementData"
-            />
-          ),
-        },
-      ];
+  );
 
-  const panels = [
-    {
-      title: t('Key details'),
-      content: <KeyDetailsInfoPanel noBorder={false} market={market} />,
-    },
-    {
-      title: t('Instrument'),
-      content: <InstrumentInfoPanel noBorder={false} market={market} />,
-    },
-    {
-      title: t('Settlement asset'),
-      content: <SettlementAssetInfoPanel market={market} noBorder={false} />,
-    },
-    {
-      title: t('Metadata'),
-      content: <MetadataInfoPanel noBorder={false} market={market} />,
-    },
-    {
-      title: t('Risk model'),
-      content: <RiskModelInfoPanel noBorder={false} market={market} />,
-    },
-    {
-      title: t('Risk parameters'),
-      content: <RiskParametersInfoPanel noBorder={false} market={market} />,
-    },
-    {
-      title: t('Risk factors'),
-      content: <RiskFactorsInfoPanel noBorder={false} market={market} />,
-    },
-    ...(market.priceMonitoringSettings?.parameters?.triggers || []).map(
-      (trigger, i) => ({
-        title: t(`Price monitoring trigger ${i + 1}`),
-        content: <MarketInfoTable noBorder={false} data={trigger} />,
-      })
-    ),
-    ...(market.data?.priceMonitoringBounds || []).map((trigger, i) => ({
-      title: t(`Price monitoring bound ${i + 1}`),
-      content: (
+  return (
+    <div>
+      <KeyDetailsInfoPanel noBorder={false} showTitle={true} market={market} />
```

Comment:
chore (blocking): boolean props such as noBorder and showTitle are a bit of a code smell to me. Rather than passing a prop just to hide show something why not pass the title it self. Its much more declarative and obvious from the component consumer what the behaviour is then.

Reply:
agree. addressed your comments. could you take another look?